signer/tests/integration/transaction_coordinator.rs

```diff
@@ -2132,20 +2126,20 @@ async fn sign_bitcoin_transaction_multiple_locking_keys() {
     // Now we check that each database has the sweep transaction and is
     // recognized as a signer script_pubkey.
     for (_, db, _, _) in signers.iter() {
-        let tx = sqlx::query_scalar::<_, BitcoinTx>(
+        let script_pubkey = sqlx::query_scalar::<_, model::ScriptPubKey>(
             r#"
-            SELECT tx
-            FROM sbtc_signer.transactions
+            SELECT script_pubkey
+            FROM sbtc_signer.bitcoin_tx_outputs
             WHERE txid = $1
+              AND output_type = 'signers_output'
             "#,
         )
         .bind(txid.to_byte_array())
-        .fetch_one(db.pool())
+        .fetch_one(ctx.storage.pool())
```

Comment:
qq: why not using `db`?

Reply:
Lots of sloppiness here, I copy pasted this and forgot to change some things back.

Reply:
https://github.com/stacks-network/sbtc/pull/1559/commits/746a980b1c12269d97740d518ae077321703a83f